equinity-historical-data/src/main/java/uk/gov/justice/laa/crime/equinity/historicaldata/mapper/Crm7Mapper.java

```diff
@@ -241,12 +239,21 @@ public interface Crm7Mapper extends CrmMapper {
     @Mapping(target="cost", source="cfc_cw_calls_costs")
     Crm7CommunicationsCostsTableDTO getClaimOfCostOfficeCallsDTOFromModel(Crm7DetailsModel model);
 
-    @Mapping(target="disbursements", expression="java(null)")
+    @Mapping(target="disbursements", source="disbursements.table")
     @Mapping(target="totals", expression="java(getDisbursementTotalsDTOFromModel(model))")
     @Mapping(target="invoiceAttachments", expression="java(convertToEnum(Crm7DisbursementDTO.InvoiceAttachmentsEnum.class, model.getCfc_disb_attach_method()))")
     @Mapping(target="officeUse", expression="java(getDisbursementOfficeDTOFromModel(model))")
     Crm7DisbursementDTO getDisbursementDTOFromModel(Crm7DetailsModel model);
 
+    @Mapping(target="disbursement", source="disbursement")
+    @Mapping(target="details", source="details")
+    @Mapping(target="miles", expression="java(null)")
```

Comment:
Can we wait for the mapping for miles?

Reply:
it might take too long. I think it is better to have a table with 1 column missing that having the whole missing table. Also consider, holding this will affect all cases that does not include car travel milage